Makefile.am

```diff
@@ -1,4 +1,5 @@
 ## Process this file with automake to produce Makefile.in
+ACLOCAL_AMFLAGS = -I m4
```

Comment:
What was the warning?  What does this do?

Reply:
Run `autoreconf -vi` in the sources root to build autoconf files. You will see libtool warning.
See also: https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.72/html_node/Input.html#index-AC_005fCONFIG_005fMACRO_005fDIR-1

Without this flag automake before 1.13 will not work properly. Not really relevant for modern automake, but still produces the warning.

Reply:
Hmmm, yep:

```
libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
```